backend/catalogo_arqueologico/piezas/models.py

```diff
@@ -145,15 +245,31 @@ class Artifact(models.Model):
 
 class TagsIds(models.Model):
     """
-    Auxiliary table to store the relationship between the tag and the artifact
-    Tag and ArtifactId must be unique together
+    Auxiliary table to store the relationship between the tag and the artifact.
```

Comment:
I think it would be a good idea to mention that these tables are used only when importing new data